src/warnet/client.py

```diff
@@ -58,18 +58,18 @@ def get_messages(src_index: int, dst_index: int):
     messages.sort(key=lambda x: x["time"])
     return messages
 
-def stop_network(network: str = "warnet"):
```

Comment:
I think this re-limits us to one warnet per docker host, which is something other said we didn't want to limit ourselves to...?

Reply:
ok ill revert this and make the necessary changes elsewhere

Reply:
Ok. I did rebase the dns seed on this and @okjodom may be rebased now too, so try and be nice 😋